test/framework/fixtures/expectFixture.go

```diff
@@ -114,7 +114,8 @@ func MakeExpectTest(t *testing.T) *ExpectFixture {
 // Run Process all expect script files with suffix Test.exp within the current directory
 func (ef *ExpectFixture) Run() {
 	disabledTest := map[string]bool{
-		"pingpongTest.exp": true,
+		"pingpongTest.exp":                    true, // broken
+		"listExpiredParticipationKeyTest.exp": true, // flaky
```

Comment:
I think it would be better to modify the expect test itself to return right away - so that the "disable" is on the test itself and we don't need to maintain "good test/bad test" lists.

Reply:
The problem with this expected does not prints out output when it is successful so there is no way to know it did not run. With the current approach tho skipped files do not appear in `go test -v` log. Plus I'll add logging here to emphasize it was skipped.

Reply:
What if the expect would output a "Skip" output ? we're already reading the output in the go-code, so we can just call the t.Skip in that case.